### ISSUE TITLE
[Sugestion] Blockbench splitting cubes

### ISSUE THREAD

**Issue by Yuuki-kito** [JannisX11/blockbench#566]
As the title says, please add a feature or a plugin to make 1 cube split into several cubes. it would really helpful for fixing bigger UV's because i struggle alot with UV Mapping on Bedrock Legacy Model. It would be really really powerful plugin/feature
![save](https://user-images.githubusercontent.com/62210133/77252817-034e3d80-6c89-11ea-8d47-7d7be52360b0.png)

**Comment by JannisX11** [comment]
Just duplicate the cube and resize each copy to be half as big. I really don't think this needs an extra tool.

**Comment by ViewPointGames** [comment]
Edit: Nevermind, I just did not know where to access that function.

I do think this should be very much a feature. I could use it pretty much right now:

<img width="982" height="568" alt="Image" src="https://github.com/user-attachments/assets/9a143c4f-055d-4dfa-8eaa-9aaf1aa78db0" />

Here I am trying to make a low poly ship, but I need the insides to be visible. I started by approximating the shape of the hull to know the constraints Ill have to work with, and now I tried to cut it in half like a bread roll along the horizontal half, to inset the top face into the cuboid and then add walls and stuff. Without this feature, this workflow is impossible to recreate, and considering that the shape I showed in the image above, I don't know how to elegantly replicate that starting with multiple meshes.
